cmd/cobra.go

```diff
@@ -119,7 +119,7 @@ func caddyCmdToCobra(caddyCmd Command) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   caddyCmd.Name,
 		Short: caddyCmd.Short,
-		Long:  caddyCmd.Long,
+		Long:  caddyCmd.Usage + "\n\n" + caddyCmd.Long,
```

Comment:
Line breaks don't look necessary because "Long" strings already include them.

```suggestion
		Long:  caddyCmd.Usage + caddyCmd.Long,
```